From 234cf96d396d401b02ca0274607a1208780525f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Sat, 22 Apr 2017 00:49:16 +0200
Subject: apps-menu: Remove excess parameters

Someone mixed up add() and add_actor() - this has been present since the
the big rewrite based on the AxeMenu extension in commit 9211fa4409d, so
there's little point in coming up with a replacement for something that
never had any effect to begin with ...
---
 extensions/apps-menu/extension.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extensions/apps-menu/extension.js b/extensions/apps-menu/extension.js
index 5b01411..a606652 100644
--- a/extensions/apps-menu/extension.js
+++ b/extensions/apps-menu/extension.js
@@ -623,7 +623,7 @@ const ApplicationsButton = new Lang.Class({
         this.applicationsBox = new St.BoxLayout({ vertical: true });
         this.applicationsScrollBox.add_actor(this.applicationsBox);
         this.categoriesBox = new St.BoxLayout({ vertical: true });
-        this.categoriesScrollBox.add_actor(this.categoriesBox, { expand: true, x_fill: false });
+        this.categoriesScrollBox.add_actor(this.categoriesBox);
 
         this.mainBox.add(this.leftBox);
         this.mainBox.add(this._createVertSeparator(), { expand: false, x_fill: false, y_fill: true});
-- 
cgit v0.12

